components/fd/component.go

```diff
@@ -294,6 +326,9 @@ type checkResult struct {
 	// Set to true if the file descriptor limit is supported.
 	FDLimitSupported bool `json:"fd_limit_supported"`
 
+	// ZombieProcesses is the number of zombie processes.
+	ZombieProcesses int `json:"zombie_processes"`
```

Comment:
the fd component looks like an OS component..  shall we move the fd component to the os?

Reply:
sure